src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -1786,8 +1786,8 @@ internal struct CENTRAL_ACCESS_POLICY
             uint nAclLength,
             uint dwAclRevision);
 
-        [DllImport("ntdll.dll", CharSet = CharSet.Unicode)]
-        internal static extern uint RtlAddScopedPolicyIDAce(
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
```

Comment:
Shall we use the API set DLL?
See http://onecoresdk/APISets/api-ms-win-security-base-l1-2-0.htm